rubix-bookkeeper/src/main/java/com/qubole/rubix/bookkeeper/BookKeeperServer.java

```diff
@@ -138,11 +152,13 @@ public void stopServer()
       log.error("Metrics reporters could not be closed", e);
     }
     server.stop();
+    log.debug("Bookkeeper Server Stopped");
```

Comment:
Suggestion: Mirror the phrasing of the start log line
```suggestion
    log.debug("Stopped BookKeeperServer");
```